misc/userscripts/qute-pass

```diff
@@ -140,9 +159,13 @@ def find_pass_candidates(domain, unfiltered=False):
             split_path = pass_path.split(os.path.sep)
             for secret in secrets:
                 secret_base = os.path.splitext(secret)[0]
-                if not unfiltered and domain not in (split_path + [secret_base]):
+                idna_domain = idna_encode(domain)
```

Comment:
This step should go outside the `for path, directories, file_names in os.walk(arguments.password_store, followlinks=True):` loop at line 152, as the domain doesn't change between entries in my password store.